independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#007ee5",

```

Comment:
@Kiveshan all independent publisher must use "#da3b01" for their icon brand color. Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json)